packages/core/src/auth/api/config-api.js

```diff
@@ -20,7 +20,7 @@ import {axiosApiProvider} from '../../apiProviders';
 
 export async function getBaseConfig() {
   return axiosApiProvider.get({
-    url: '/ws/rest/com.axelor.apps.base.db.AppBase/',
+    url: '/ws/rest/com.axelor.studio.db.AppBase/',
```

Comment:
You forgot to use the router here. I updated your branch, can you check that all API requests are taken care of please ? It seems that a few are still missing. I saw '/ws/rest/com.axelor.apps.base.db.App/search' and '/ws/rest/com.axelor.apps.base.db.AppMobileSettings/' but maybe there is more.